test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -238,10 +238,8 @@ def initialize(grpc_stub)
 
     def api_client
       ssl_creds = GRPC::Core::ChannelCredentials.new
-      authentication = Google::Auth.get_application_default(LOGGING_SCOPE)
-      creds = GRPC::Core::CallCredentials.new(lambda do |a_hash|
-        authentication.apply(a_hash, use_configured_scope: true)
-      end)
+      authentication = Google::Auth.get_application_default
+      creds = GRPC::Core::CallCredentials.new(authentication.updater_proc)
       ssl_creds.compose(creds)
```

Comment:
Why isn't this assigned like it is above? `creds = ssl_creds...`

Reply:
This was present in the original test code, which basically means the test wasn't doing the right thing, but unrelated to this rollback.